modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -380,6 +380,39 @@ impl LightningClientModule {
         Ok(operation_id)
     }
 
+    async fn get_next_operation_id(
```

Comment:
Nit: could use a comment. I'm a bit confused what is it for, without looking at the context and/or documentation first.